exist-core/src/main/java/org/exist/xquery/functions/fn/FunBaseURI.java

```diff
@@ -21,67 +21,55 @@
  */
 package org.exist.xquery.functions.fn;
 
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.StringUtils;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.exist.xquery.*;
```

Comment:
@dizzzz I find it hard to disagree with avoiding unused imports here

Reply:
Please could you elaborate, I find this text a bit cryptic (I really can't understand it). 

the  `import org.exist.xquery.*;` is really needed?

Reply:
as in: there are no unused imports